src/lib/relicFilters.js

```diff
@@ -85,14 +85,28 @@ export const RelicFilters = {
     out.push(...relics.filter(x => x.part == Constants.Parts.Feet).filter(x => request.mainFeet.length == 0 || request.mainFeet.includes(x.main.stat)))
     out.push(...relics.filter(x => x.part == Constants.Parts.PlanarSphere).filter(x => request.mainPlanarSphere.length == 0 || request.mainPlanarSphere.includes(x.main.stat)))
     out.push(...relics.filter(x => x.part == Constants.Parts.LinkRope).filter(x => request.mainLinkRope.length == 0 || request.mainLinkRope.includes(x.main.stat)))
-  
+
     return out;
   },
 
   applyEnhanceFilter: (request, relics) => {
     return relics.filter(x => x.enhance >= request.enhance);
   },
 
+  applyEquippedFilter: (request, relics) => {
```

Comment:
actual filter logic here.  DB/store relics set from import is missing foreign key to character.equipped.   e.g., all the relics "equippedBy" IDs are `undefined`.

i had to manually create the blacklist here by iterating over all character.equipped maps.

maybe a backlog bug to repair metadata to ensure links are maintained during import?

Reply:
https://github.com/fribbels/hsr-optimizer/issues/56